install/kubernetes/tetragon/values.yaml

```diff
@@ -64,9 +64,11 @@ tetragon:
   securityContext:
     privileged: true
   # -- Overrides the default livenessProbe for the tetragon container.
-  livenessProbe: {}
-  #  grpc:
-  #    port: 54321
+  livenessProbe:
+    timeoutSeconds: 60
+    grpc:
+      port: 6789
+      service: "liveness"
```

Comment:
maybe keep this value as is and update https://github.com/cilium/tetragon/blob/4c47837a06f473b45ad189149c63853f2f502304/install/kubernetes/tetragon/templates/_container_tetragon.tpl#L70-L81 instead just to follow the same pattern as other values like these => https://github.com/cilium/tetragon/blob/4c47837a06f473b45ad189149c63853f2f502304/install/kubernetes/tetragon/values.yaml#L57-L63)

Reply:
Makes sense. I have also done this update.